src/commands/createStaticWebApp/createStaticWebApp.ts

```diff
@@ -17,7 +18,18 @@ import { IStaticWebAppWizardContext } from './IStaticWebAppWizardContext';
 import { postCreateStaticWebApp } from './postCreateStaticWebApp';
 import { setWorkspaceContexts } from './setWorkspaceContexts';
 
+export const isVerifyingWorkspaceEmitter: EventEmitter<boolean> = new EventEmitter<boolean>()
+let _isVerifyingWorkspace: boolean = false;
+
+isVerifyingWorkspaceEmitter.event(e => _isVerifyingWorkspace = e);
```

Comment:
I like the use of EventEmitter here. Howevever, this returns a `Disposable`. Do we have to dispose it?

Reply:
I think we're okay here.  I don't see this in any of our other extensions that use `EventEmitters`.  I think all of this gets disposed when the extension closes and we wouldn't want to dispose it earlier than that anyway.

Reply:
> I don't see this in any of our other extensions that use EventEmitters

Uhh where? We should be disposing... 😬 

Sounds like you want to add it to `ext.context.subscriptions`

>     export interface ExtensionContext {
> 
>         /**
>          * An array to which disposables can be added. When this
>          * extension is deactivated the disposables will be disposed.
>          */
>         readonly subscriptions: { dispose(): any }[];

Reply:
https://github.com/microsoft/vscode-azuretools/blob/ba49322b5120c25f9202f0edd3383097afbab58d/ui/src/openReadOnlyContent.ts#L55

But okay, I was thinking about doing that as well, just didn't really want to expose the _verifyingWorkspace variable outside of the file, but I guess it doesn't really matter that much.

Reply:
Mmm yeah we're not great at disposing the `EventEmitter` itself (we should, tho). We are pretty good at disposing the `Event` which I think what Alex's comment was about